code/modules/hydroponics/grown.dm

```diff
@@ -117,11 +119,18 @@
 	if(seed)
 		for(var/datum/plant_gene/trait/trait in seed.genes)
 			trait.on_squash(src, target)
-
-	reagents.reaction(T)
-	for(var/A in T)
-		reagents.reaction(A)
-
+	if(!seed.get_gene(/datum/plant_gene/trait/noreact))
+		reagents.reaction(T)
+		for(var/A in T)
+			reagents.reaction(A)
+		qdel(src)
+	if(seed.get_gene(/datum/plant_gene/trait/noreact))
+		playsound(loc, 'sound/effects/fuse.ogg', 75, 1, -3)

```

Comment:
No lol. It's a plant dude.

Reply:
it needs some form of sharp auditory feedback when it's primed. Fuse is more organic than the grenade beeps, which combustible lemon uses for some reason, but still designed for this purpose. Reagent bubbling and the backpack rustle were two more sounds i thought of that could be used, though.

Reply:
You realize that it's just a plant getting squashed on the ground right.

Reply:
see below on using in hand. a bit of auditory feedback is good regardless, though